### DISCUSSION TITLE
Duplicator Tree Not Working

### DISCUSSION THREAD

**Discussion by ninjatwinkie1** [itzg/docker-minecraft-server#2165] [Q&A]
### Describe the problem

I enabled the duplicator tree in config file by making the value 1 instead of 0. I have projecte so I used a watch of flowing time on a dark pedestal to speed up the process and the only thing that happended was the apples dropping.

My setup in case you need it ;
![Minecraft 1 7 10 5_31_2023 12_10_32 PM](https://github.com/itzg/docker-minecraft-server/assets/76224830/bc0193ec-53e3-4abb-b2bd-881301a96221)

### Container definition

```
Paste run command or compose file here
```

### Container logs

```
Paste logs here
```

**Comment by itzg** [comment]
I'm not sure how this is related to the image itself.

**Comment by ninjatwinkie1** [comment]
I left the image of my setup in case there was a problem with it

**Comment by itzg** [comment]
I'm talking about this **Docker image** not some screenshot image.  Please provide **container** setup (compose file or docker run command line), which file you modified, and any relevant logs.
